pages/api/hashicorp.ts

```diff
@@ -15,9 +17,50 @@ export default function handler(
 	res: NextApiResponse<ResponseData>
 ) {
 	const { query } = req
-	const searchParam = query.search || ''
+	const searchParam = (query.search as string) || ''
+	const hideNoPicture = (query.hideNoPicture as string) || ''
 
-	// Sr. candidate TODO: Perform DB query and return the result
+	try {
+		const db = new Database('hashicorp.sqlite')
+		const isDepartmentId = !isNaN(Number(searchParam))
+		let statement = null
 
-	res.status(200).json({ results: [] })
+		if (searchParam === '') {
```

Comment:
This is a pretty big "if" statement, could we possibly remove it?

Reply:
Yes, done!